content/rs/clusters/logging/log-security.md

```diff
@@ -16,48 +16,68 @@ aliases: [
 ---
 Redis Enterprise comes with [a set of logs]({{< relref "/rs/clusters/logging/_index.md" >}}) on the server and available through the user interface to assist users in investigating actions taken on the server and to troubleshoot issues.
 
-## Sending logs to a remote logging server
+## Send logs to a remote logging server
 
 Redis Enterprise sends logs to syslog by default. You can send these logs to a remote logging server by configuring syslog.
 
 To do this, modify the syslog or rsyslog configuration on your operating system to send logs in `/var/opt/redislabs/log` to a remote monitoring server of your choice.
 
 ## Log rotation
 
-Redis Enterprise uses the default logrotate daemon to schedule rotation of logs stored on the operating system. The configuration of log rotation may be found at /etc/logrotate.d.
+Redis Enterprise uses the default `logrotate` daemon to schedule rotation of logs stored on the operating system. You can find the log rotation configuration file at `$pkgconfdir/logrotate.conf` as of Redis Enterprise Software version 7.2 (`pkgconfdir` is `/opt/redislabs/config` by default, but can be changed in a custom installation).
 
-By default the log rotation should occur on a daily basis. We recommend that you send log files to a remote logging server so that they can be more effectively maintained.
+By default, log rotation occurs every five minutes or when logs exceed 200 MB. We recommend sending log files to a remote logging server so you can maintain them more effectively.
```

Comment:
There's a confusion here. logrotate tool is ran every 5 minutes (by the job scheduler). The logrotate tool examines every log file and rotates each log that exceeds 200MB.

Reply:
Thanks for clarifying. That makes more sense.